fs/operations/operations_test.go

```diff
@@ -785,6 +785,7 @@ func TestMoveFile(t *testing.T) {
 	ctx := context.Background()
 	r := fstest.NewRun(t)
 	defer r.Finalise()
+	precision := fs.GetModifyWindow(ctx, r.Fremote, r.Flocal)
```

Comment:
It might save a few lines. But in most tests where the precision of both remote and local fs is needed, the precision is used more than once. I'm not sure if calling the method each time it needs the precision is a good idea.

For example, in `TestCopyFileCompareDest`, `precision` is used 10 times.